ts_app_struct/api/.gitignore

```diff
@@ -0,0 +1,20 @@
+# The directory Mix will write compiled artifacts to.
```

Comment:
It's better to move the contents of `ts_app_struct` to top level. The existing top-level contents of transportscheduler project would go into one of the three projects, namely input_parser, network, and api.